formula-android/build.gradle

```diff
@@ -5,7 +5,6 @@ apply plugin: 'kotlin-android-extensions'
 apply plugin: 'kotlin-kapt'
 apply plugin: "com.github.ben-manes.versions"
 apply plugin: "com.vanniktech.maven.publish"
-apply plugin: 'org.jetbrains.dokka-android'
```

Comment:
Errors were appearing since `com.vanniktech.maven.publish` includes this plugin already.